server/docker/entrypoint-db-setup

```diff
@@ -24,14 +24,14 @@ source /run/secrets/django || true
 }
 export SECRET_KEY="$DJANGO_SECRET_KEY"
 
-until pg_isready -h "${DATABASE_HOST}" -p "${DATABASE_PORT:-5432}"; do
+until pg_isready -h "${DATABASE_HOST:-postgres}" -p "${DATABASE_PORT:-5432}"; do
```

Comment:
This is probably more of a nit-pick than anything else.  `DATABASE_HOST` defaults to `pgbouncer` in the default configuration templates, not `postgres`.  Ideally, that default would be matched here.

However, I am aware that `docker-compose.dev.yml` diverges and uses `postgres`.  Since `postgres` is also an alias for `pgbouncer` in the other configuration templates, it should be alright.

Reply:
Done